metron-platform/metron-common/pom.xml

```diff
@@ -383,6 +383,10 @@
                                     <pattern>com.google.common</pattern>
                                     <shadedPattern>org.apache.metron.guava.${guava_version}</shadedPattern>
                                 </relocation>
+                                <relocation>
+                                    <pattern>com.google.thirdparty</pattern>
+                                    <shadedPattern>org.apache.metron.guava.thirdparty.${guava_version}</shadedPattern>
```

Comment:
This fixes an error with `SimpleFlatFileSummarizerTest.testWholeFile` caused by how we relocate Guava. 
 We relocate `com.google.common` which does not include all packages within Guava.  It specifically misses `com.google.thirdparty.publicsuffix.PublicSuffixPatterns` which is the root cause of this test failure.

We may want to ensure that we relocate everything under `com.google` everywhere, but I didn't want to make such a drastic change just yet.  And I know @merrimanr has a PR out that may shuffle what we need to do here.